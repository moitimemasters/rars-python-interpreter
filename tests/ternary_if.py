a = 5
1 if a + 3 > 10 else 0 if a + 5 > 10 else 1 if a + 6 > 10 else 0
