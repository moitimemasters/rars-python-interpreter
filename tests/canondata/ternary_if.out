file: tests/ternary_if.py
started lexing
ended lexing
started parsing ast
Error: Expected ":" after condition
ended parsing ast
started compiling
ended compiling
compiled instructions:
	 Instruction: 2
	 Instruction: 8
	 Instruction: 0
	 Instruction: 2
	 Instruction: 7
	 Instruction: 2
	 Instruction: 7
	 Instruction: 6
	 Instruction: 2
	 Instruction: 5
	 Instruction: 2
started interpreting
ended interpreting
Interpretation result: 0
